opm/autodiff/FlowMainEbos.hpp

```diff
@@ -559,12 +624,24 @@ namespace Opm
             // create output writer after grid is distributed, otherwise the parallel output
             // won't work correctly since we need to create a mapping from the distributed to
             // the global view
-            output_writer_.reset(new OutputWriter(grid(),
+            std::vector<double> global_perms;
+            const double* writer_perms = fluidprops_->permeability();
+
+            if ( must_distribute_ )
```

Comment:
hm, ok. In this case, would you mind removing the `must_distribute_` attribute and replace it by `grid().comm().size() > 1`?